### ISSUE TITLE
Can I use an AMD GPU to train Yolo4? 

### ISSUE THREAD

**Issue by runaway** [AlexeyAB/darknet#5556]
Hi, AlexeyAB

I have an AMD GPU and I want to know if I can use it to accelerate training Yolo4 model? 

Thank you

**Comment by tloki** [comment]
Since darknet is written in C/C++ using CUDA, I don't think so.

**Comment by mtlcrusher** [comment]
yes you can, but you need to manually add all the kernels for amd gpu and add some line in darknet.cpp to use your customized file
